core/src/main/java/eu/kevin/core/plugin/Kevin.kt

```diff
@@ -1,18 +1,30 @@
 package eu.kevin.core.plugin
 
+import java.util.*
 import java.util.concurrent.atomic.AtomicBoolean
 
 object Kevin {
 
     private val configurationLock = AtomicBoolean(false)
     private val plugins = mutableSetOf<KevinPlugin>()
+    private var locale: Locale? = null
```

Comment:
maybe let's not make this optional and just preset it to be English?

Reply:
if it's not set, it would use system locale, which might be what user wants

Reply:
oh, ok